### PR TITLE
Feat/new eth routes

### PR DESCRIPTION
**Pull request by mifeng**
This PR added new routes for `allowances` and `balances` that separates the input params into 2 arrays for addresses and decimals, rather than a key-value dict. This makes it much easier to to test the API with Postman.

Since incorporating this change would require a similar change in the client and we should discuss this change internally, I'm marking this as Draft.

### CODE REVIEW

File: src/services/eth.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default class Ethereum {
   }
 
   // get ERC-20 token allowance
-  async getERC20Allowance (wallet, spender, tokenAddress, decimals) {
+  async getERC20Allowance (wallet, spender, tokenAddress, decimals = 18) {
```
Comment (vic-en) [inline src/services/eth.js:54]:
```suggestion
  async getERC20Allowance (wallet, spender, tokenAddress, decimals) {
```

File: src/services/eth.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default class Ethereum {
   }
 
   // get ERC-20 token balance
-  async getERC20Balance (wallet, tokenAddress, decimals) {
+  async getERC20Balance (wallet, tokenAddress, decimals = 18) {
```
Comment (vic-en) [inline src/services/eth.js:40]:
```suggestion
  async getERC20Balance (wallet, tokenAddress, decimals) {
```

**Review by vic-en — APPROVED** [review]
LGTM

**Review by sdgoh — APPROVED** [review]
Verified working for Hummingbot new ETH route update

**Pull request merged**